runtime/compiler/runtime/J9CodeCacheManager.cpp

```diff
@@ -297,6 +314,13 @@ J9::CodeCacheManager::allocateCodeCacheSegment(size_t segmentSize,
    codeCacheSizeToAllocate = (codeCacheSizeToAllocate + (vmemParams.pageSize-1)) & (~(vmemParams.pageSize-1));
    vmemParams.byteAmount = codeCacheSizeToAllocate;
 
+   if (!self()->isSufficientPhysicalMemoryAvailableForAllocation(codeCacheSizeToAllocate))
+      {
+      if (config.verboseCodeCache())
+         TR_VerboseLog::writeLineLocked(TR_Vlog_CODECACHE, "Failed to allocate code cache segment due to low memory");
```

Comment:
Suggestion: "...due to low physical memory"

Reply:
Another suggestion: I think it may be useful for the user to see how much we wanted to allocate and hoe much memory was available.